app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Users < ApplicationController
```

Comment:
- [ ] Please rename `Users` to `UsersController` to fix the superclass mismatch for class `Users` bug 🐛 
![image](https://github.com/leloufadel/my_blog_app/assets/46839548/9dc9b72f-eadb-4616-982d-7033f1252142)